package.json

```diff
@@ -45,11 +43,12 @@
     "@commitlint/config-conventional": "19.2.2",
     "@defi-wonderland/natspec-smells": "1.1.1",
     "@defi-wonderland/smock-foundry": "1.5.0",
+    "ds-test": "https://github.com/dapphub/ds-test.git#e282159",
```

Comment:
why adding this package?

Reply:
when using

- node v18.20.2
- npm  10.8.1
- yarn 1.22.22

and doing a clean clone of the `dev` branch, I get the following error

    Error: Error while compiling contracts: Error: Couldn't find node_modules/ds-test/src/test.sol
        at /home/user/wonderland/foo/node_modules/@defi-wonderland/smock-foundry/dist/utils.js:71:19
        at Generator.throw (<anonymous>)
        at rejected (/home/user/wonderland/foo/node_modules/@defi-wonderland/smock-foundry/dist/utils.js:6:65)

since `ds-test` is explicitly listed in `remappings.txt`: 

    ds-test/=node_modules/ds-test/src

but the directory isn't there:

    [I] ~/w/foo (dev)[1]
    > ls node_modules/ds-test
    ls: cannot access 'node_modules/ds-test': No such file or directory

so I added it as an explicit dependency to ensure the files are found.

perhaps we're using a different node/npm/yarn that has a different handling of peer dependencies or organizes files differently in `node_modules`?

Reply:
> since ds-test is explicitly listed in remappings.txt

this is because `forge-std` imports `ds-test` and without the remapping it doesn't work

Reply:
I dropped the commit as it seems I'm the only one replicating this

- what npm/node/yarn versions are you using?
- do you find the `ds-test` files under `node_modules/ds-test`?

Reply:
using nvm 18, and nope, not finding ds-test under node_modules..